src/client/clouds.cpp

```diff
@@ -145,15 +149,18 @@ void Clouds::updateMesh()
 		// shader mixes the base color, set via EmissiveColor
 		c_top_f = c_side_1_f = c_side_2_f = c_bottom_f = video::SColorf(1.0f, 1.0f, 1.0f, 1.0f);
 	}
-	c_side_1_f.r *= 0.95f;
-	c_side_1_f.g *= 0.95f;
-	c_side_1_f.b *= 0.95f;
-	c_side_2_f.r *= 0.90f;
-	c_side_2_f.g *= 0.90f;
-	c_side_2_f.b *= 0.90f;
-	c_bottom_f.r *= 0.80f;
-	c_bottom_f.g *= 0.80f;
-	c_bottom_f.b *= 0.80f;
+	video::SColorf shadow = m_params.color_shadow;
+
+	c_side_1_f.r *= shadow.r * 0.33f + 0.66f;
+	c_side_1_f.g *= shadow.g * 0.33f + 0.66f;
+	c_side_1_f.b *= shadow.b * 0.33f + 0.66f;
+	c_side_2_f.r *= shadow.r * 0.66f + 0.33f;
+	c_side_2_f.g *= shadow.g * 0.66f + 0.33f;
+	c_side_2_f.b *= shadow.b * 0.66f + 0.33f;
+	c_bottom_f.r *= shadow.r;
+	c_bottom_f.g *= shadow.g;
+	c_bottom_f.b *= shadow.b;
```

Comment:
with the default of 204 this results in 0.92, 0.85, 0.8 (respectively)
which is different than before.

so now I have to ask: where do these magic numbers come from? what is the effect of the change?

Reply:
^ this was not answered

Reply:
After I changed the "magic numbers" it should produce the same colors. This change is to generalize the color calculation to all possible shadow colors.